README.md

```diff
@@ -54,7 +54,29 @@ export PORT=3000
 npm run build && npm run start
 ```
 
-### Tests
+## Tooling Dev Experience
+
+### Commitlint
+
+Nous utilisons [commitlint](https://commitlint.js.org/#/guides-local-setup) pour vérifier que les messages de commit suive bien la convention ce dernier nécessite l'installation de husky.
+
+```
+npx husky install
+```
+
+### Eslint
+
+Afin d'améliorer votre expérience de dev vous pouvez utiliser [l'extenstion Eslint](https://marketplace.visualstudio.com/items?itemName=dbaeumer.vscode-eslint).
+
+Ajouter la configuration ci-dessous pour fixer les problèmes de linting automatiquement
+
+```
+  "editor.codeActionsOnSave": {
+    "source.fixAll.eslint": true
```

Comment:
Does it sort json ?

I renember you also suggested `"source.fixAll.sortJSON": true`